doc/reference/cmdline.rst

```diff
@@ -247,6 +330,8 @@ Developer features
 
 .. option:: --dev <feature,feature,...,feature>
 
+    comma-separated list of features
```

Comment:
As we are modifying it, maybe add a warning above to indicate one should not use these on production (seems obvious but just in case)

Reply:
fixed